### PR TITLE
Move language selector into the user menu (dropdown) - Implemented in plonetheme.teamraum

### PR DESCRIPTION
**Pull request by maethu**
This PR requires https://github.com/4teamwork/plonetheme.teamraum/pull/563

screen shot 2017-07-19 at 16 47 37

The implementation of the language menu has been completely moved to the plonetheme.teamraum.

Closes https://github.com/4teamwork/opengever.core/issues/966

### CODE REVIEW

**Comment by maethu** [comment]
@phgross I fixed the labels
screen shot 2017-08-02 at 11 00 47

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ Changelog
 - Disable reference number column in the document listing of the inbox. [phgross]
 - Enable \*.msg download for all download copy links. [phgross]
 - Fix mail download: Convert LF to CRLF only for EML mails. [phgross]
+- Move language selector into the user menu (dropdown) - Implemented in plonetheme.teamraum. [mathias.leimgruber]
```
Comment (phgross) [inline docs/HISTORY.txt]: Wrong release ...

File: sources.cfg [hunk 2]
```diff
@@ -20,3 +21,4 @@ auto-checkout = ${buildout:development-packages}
 [branches]
 Products.LDAPUserFolder = ftw
 Products.LDAPMultiPlugins = ftw
+plonetheme.teamraum = mle-move-language-switch
```
Comment (phgross) [inline sources.cfg]: ✂️  as soon the Theme PR is merged ...

**Comment by maethu** [comment]
@phgross rebased & ready

**Pull request merged**